sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -282,3 +272,18 @@ private[hive] class SparkExecuteStatementOperation(
     }
   }
 }
+
+object SparkExecuteStatementOperation extends Logging {
+  def getTableSchema(result: DataFrame): TableSchema = {
```

Comment:
May I use `reult.queryExecution.analyzed.output` instead? I mean `Seq[Attribute]` instead of `StructType`.